cpp/src/arrow/compute/exec.h

```diff
@@ -440,5 +441,30 @@ Result<Datum> CallFunction(const std::string& func_name, const ExecBatch& batch,
 
 /// @}
 
+/// \defgroup compute-function-executor One-shot calls to obtain function executors
```

Comment:
I forgot: this new Doxygen group should be referred somewhere in the Sphinx API docs, otherwise it won't be visible.
See https://github.com/apache/arrow/blob/master/docs/source/cpp/api/compute.rst